src/lv_demo_widgets/lv_demo_widgets.py

```diff
@@ -0,0 +1,575 @@
+#!/opt/bin/lv_micropython
+import lvgl as lv
+import time
+import SDL
```

Comment:
Aren't all these imports supposed to come from `header.py`?

Reply:
Yes, normally these come from header.py. However, in initgui.py I create a 240x240 screen for the SDL driver (the t-watch also uses 240x240, while the ili9341 uses 320x240). This screen size is too small for this demo. Therefore I initialize a 480x320 screen on the SDL driver only. 
I think we can also run this demo on the ili9341 with 320x240 resolution.